dist/main.js

```diff
@@ -0,0 +1,63 @@
+(function webpackUniversalModuleDefinition(root, factory) {
```

Comment:
Oh, actually deleted all-together. I think this is from when we were messing around with the output paths.